### DISCUSSION TITLE
Revisited: Linux and windows.

### DISCUSSION THREAD

**Discussion by leamas** [opencpn-manuals/main#7] [General]
I have problems with filenames and execute permissions. It's not only me, the github  actions problems have the same issues.

I have now fixes paths in plugins/site.yml twice, changed the casing of several paths so they match the real paths in sources/. I understand that Windows doesn't care, and it works anyway there. But we need to sort this out.

Same with the execute permissions, notably on source-statet.sh -- the occasionally disappear.

All this said, as long as the github actions job passes everything is fine.

**Comment by rgleason** [comment]
Alec, I am aware of this and will try to pay attention to the issue.

**Reply by leamas** [reply]
Since we now at last have a somewhat working build process, the check is simple: Watch the logs from the Github Actions workflows avialable undet the Actions button. If there are errors like those described here, the build will fail and not be published.
